goby.go

```diff
@@ -3,29 +3,59 @@ package main
 import (
 	"flag"
 	"fmt"
-	"github.com/pkg/profile"
 	"github.com/goby-lang/goby/bytecode"
 	"github.com/goby-lang/goby/parser"
 	"github.com/goby-lang/goby/vm"
+	"github.com/pkg/profile"
 	"io/ioutil"
+	"log"
 	"os"
 	"path"
-	"strings"
-	"log"
 	"path/filepath"
+	"strings"
 )
 
+// Version stores current Goby version
+const Version string = "0.0.1"
+
+// HelpMessage tells users how to use this program
+const HelpMessage string = `
+Usage: goby [filepath]
+
+Available flags:
+
+  -c   compile target program to bytecode
+  -p   profile target program execution
+  -v   show current Goby version
+  -h   show this help message
+
+For more details, visit: https://github.com/goby-lang/goby
+`
+
 func main() {
 	compileOptionPtr := flag.Bool("c", false, "Compile to bytecode")
 	profileOptionPtr := flag.Bool("p", false, "Profile program execution")
+	versionOptionPtr := flag.Bool("v", false, "Show current Goby version")
+	helpOptionPtr := flag.Bool("h", false, "Show help message")
```

Comment:
Actually you don't have to add this, the flag package already takes care this for us. You can try `goby -h` on master branch.